setup.py

```diff
@@ -18,19 +18,22 @@
 exec(open("sdk/_version.py").read())
 
 setup(
-    name="sdk",
-    version="0.0.1",
+    name="pasqal-sdk",
+    version="0.0.3",
```

Comment:
When publishing a new version to pypi, is this the version number taken into account ? I.e. does this mean we nede to manually bump this when we want to publish a new version ?

Reply:
Right now, yes. I think we could/should use the `__version__` in `sdk/_version.py` instead like in the pulser repo.

I'll take a look at it in this PR